web-common/src/features/dashboards/pivot/pivot-expansion.ts

```diff
@@ -190,8 +183,10 @@ export function queryExpandedRowMeasureValues(
         timeFilters,
       );
 
-      // TODO: Merge filters
-      // mergeFilters(rowNestFilters, sortFilters)
+      const mergedFilter = mergeFilters(
+        createAndExpression(rowNestFilters),
+        config.whereFilter,
+      );
```

Comment:
Don't have to add this here now. We can merge with global filter later at the query level. 

Idea to add mergeFilters here was to add - 
1. Filters for a sub row
2. Filters for sorted column

I can take care of this outside of this PR.